workshops/1.md

```diff
@@ -102,7 +102,7 @@ Finally we need to *always* unit test the API calls, so far we haven't had a goo
 
 ### Collaboration with designers
 
-We need to be aware of anti-patterns, UI choices that are too fragile and could easily break with the next iOS update. Diverging needlessly from the platform language. Adding too much visual fluff that takes a lot of time to develop and has very little pay-off. We work with a lot of startups and companies that will maintain our code in the future, so it's important for us to keep things in the native side as much as possible, to avoid future issues for them. In other words, if it involves a hack or going against the platform APIs, then it's probably a bad idea.
+We need to be aware of anti-patterns, UI choices that are too fragile and could easily break with the next iOS update. Diverging needlessly from the platform language. Adding too much visual fluff that takes a lot of time to develop and has very little pay-off. (These sentences before are a bit vague to me, are they summing up what anti-patterns are or defining it? I think it should be reworded) We work with a lot of startups and companies that will maintain our code in the future, so it's important for us to keep things in the native side as much as possible, to avoid future issues for them. In other words, if it involves a hack or going against the platform APIs, then it's probably a bad idea.
```

Comment:
Isn't diverging needlessly from the platform language a definition of what an anti pattern is? 🤔

Otherwise I owuld suggest changing it into this: 

There are a couple of things that we should be aware of in collaborating with designers:
- The use of anti-patterns 
- UI choices that are too fragile and could easily break with the next iOS update 
- Diverging needlessly from the platform language 
- Adding too much visual fluff that takes a lot of time to develop and has very little pay-off.